README.md

```diff
@@ -1,69 +1,289 @@
+[![Stars](
+https://img.shields.io/github/stars/ESIPFed%2FGeoweaver?style=for-the-badge
+)](https://github.com/ESIPFed/Geoweaver/stargazers) 
+[![Forks](
+https://img.shields.io/github/forks/ESIPFed/Geoweaver?style=for-the-badge&color=%23f2b40a
+)](https://github.com/ESIPFed/Geoweaver/network/members)
+[![Issues](
+https://img.shields.io/github/issues/ESIPFed/Geoweaver?style=for-the-badge&color=%2363c1ff
+)](https://github.com/ESIPFed/Geoweaver/issues) [![Coverage](
+https://img.shields.io/codecov/c/github/ESIPFed/Geoweaver?style=for-the-badge
+)](https://codecov.io/)
+![PyPi](https://img.shields.io/pypi/v/pygeoweaver?style=for-the-badge) 
+![Minimum Java Version](https://img.shields.io/badge/Java-11%2B-%23ed8b02?style=for-the-badge&logo=openjdk
+)
+![Geoweaver Docs](https://img.shields.io/badge/Docs-Geoweaver-%23c4ff7d?style=for-the-badge&logo=readthedocs&link=https%3A%2F%2Fgeoweaver.dev%2F
+)
+
+![logo](src/main/resources/static/img/geoweaver-readme-banner.png)
+
+
+### GitHub Repo: https://github.com/ESIPFed/Geoweaver 
+### Pygeoweaver (Python Bindings): https://github.com/ESIPFed/pygeoweaver
+### Live Demo: https://geobrain.csiss.gmu.edu/Geoweaver
+
 
-# [Geoweaver](https://esipfed.github.io/Geoweaver/)
 
 Geoweaver is an in-browser software allowing users to easily compose and execute full-stack data processing workflows via taking advantage of online spatial data facilities, high-performance computation platforms, and open-source deep learning libraries. It provides all-in-one capacity covering server management, code repository, workflow orchestration software, and history recorder. 
 
 It can be run from both local and remote (distributed) machines.
 
-Why choose Geoweaver?
+# Why choose Geoweaver?
+
 1) Safely Store all your progress along the way.
 2) Stay organised and productive through out your years-long research
 4) Seamlessly connect to external servers with SSH.
 5) In-Built Web UI with full support for Python.
 
+For further insights into Geoweaver, please explore the website at https://geoweaver.dev.
 GeoWeaver is a community effort. Any contribution is welcome and greatly appreciated! 
 
-# Software Goals
+# Features
+
+1) **Host Management**:
+  - Register machines via SSH as hosts for running processes.
+  - Add Jupyter Servers as host resources for interaction and workflow editing.
+
+2) **Process Variety**:
+  - Add various types of processes, such as bash scripts for data downloading.
+
+3) **Jupyter Notebook Integration**:
+  - Upload or import Jupyter notebooks.
+  - Intercept websocket traffic to save notebook versions, enabling easy revision history access.
+
+4) **Process History and Logging**:
+  - Detailed history of every process run, including logs and outputs, is stored.
+
+5) **Workflow Management**:
+  - Link processes to create workflows for parallel or sequential execution across different resources.
+  - All aspects of workflow management are centralized within GeoWeaver.
+
+# Geoweaver Installation Guide
```

Comment:
please keep this installation guide as usual. Don't add these detailed steps in the front page. When people are interested, they should click on the install guide link which was here.